streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -266,6 +266,10 @@
     public static final String ZOOKEEPER_CONNECT_CONFIG = "zookeeper.connect";
     private static final String ZOOKEEPER_CONNECT_DOC = "Zookeeper connect string for Kafka topics management.";
 
+    /** {@code replication.factor} */
+    public static final String MIN_IN_SYNC_REPLICAS_CONFIG = "min.insync.replicas";
```

Comment:
This is one of many topic configs. Would it be better to introduce a prefix and take any configs that have that prefix?